venus.json

```diff
@@ -0,0 +1,28 @@
+{
+	"response": [
+		{
+			"maintainer": "Flower Sea",
+			"oem": "Xiaomi",
+			"device": "Xiaomi 11",
+			"filename": "crDroidAndroid-14.0-20231113-venus-v10.0-BETA.zip",
+			"download": "https://sourceforge.net/projects/crdroid/files/venus/10.x/crDroidAndroid-14.0-20231113-venus-v10.0-BETA.zip/download",
+			"timestamp": 1699861984,
+			"md5": "90b6f9139d62cd8c1102ae1d052afcb3",
+			"sha256": "bd914b2719606b582c1d5e4e2783ef0df04326eb9fbe447b8741e72db7e45d8c",
+			"size": 1318327389,
+			"version": "10.0",
+			"buildtype": "Beta",
+			"forum": "",
+			"gapps": "https://github.com/MindTheGapps/14.0.0-arm64/releases",
+			"firmware": "",
+			"modem": "",
+			"bootloader": "",
+			"recovery": "https://sourceforge.net/projects/crdroid/files/venus/10.x/recovery/",
+			"paypal": "",
+			"telegram": "https://t.me/mi11dev",
+			"dt": "https://github.com/crdroidandroid/android_device_xiaomi_venus.git",
```

Comment:
No need to add ".git"
https://github.com/crdroidandroid/android_device_xiaomi_venus